src/main/scala/framework/EitherUtilities.scala

```diff
@@ -0,0 +1,9 @@
+package framework
```

Comment:
This casts a Boolean to an Either, with a suitable string fail message for the left portion. I use it in AuthorisationService as part of a for comprehension, I think it's quite a nice technique, so nice that I've almost certainly reinvented the wheel.